trackers.json

```diff
@@ -69,7 +69,7 @@
     {"Name": "The Horror Charnel", "Abbreviation": "THC", "Type": "Movie/TV", "Codebase": "-", "Users": "-", "Torrents": "-", "Peers": "-", "Ratio": "Yes", "Ratio Diff": "-", "Points": "Yes", "Birthdate": "-", "Join": "-", "Join Diff": "-", "Updated": "12/29/19"},
     {"Name": "ThePlace", "Abbreviation": "TP", "Type": "Self-help/PUA", "Codebase": "TCG", "Users": "-", "Torrents": "64,567", "Peers": "-", "Ratio": "Yes", "Ratio Diff": "Hard", "Points": "No", "Birthdate": "-", "Join": "Application/Invite", "Join Diff": "-", "Updated": "12/30/19"},
     {"Name": "TorrentBD", "Abbreviation": "TBD", "Type": "General", "Codebase": "-", "Users": "123,467", "Torrents": "243,742", "Peers": "-", "Ratio": "Yes", "Ratio Diff": "Easy", "Points": "Yes", "Birthdate": "2010", "Join": "Invite/Recruitment", "Join Diff": "Easy", "Updated": "12/31/19"},
-    {"Name": "TorrentLeech", "Abbreviation": "TL", "Type": "General", "Codebase": "-", "Users": "-", "Torrents": "-", "Peers": "-", "Ratio": "-", "Ratio Diff": "-", "Points": "-", "Birthdate": "-", "Join": "-", "Join Diff": "-", "Updated": "12/29/19"},
+    {"Name": "TorrentLeech", "Abbreviation": "TL", "Type": "General", "Codebase": "Gazelle", "Users": "-", "Torrents": "1,061,375", "Peers": "-", "Ratio": "Yes", "Ratio Diff": "Medium", "Points": "Yes", "Birthdate": "2005", "Join": "Invite/Open", "Join Diff": "Easy", "Updated": "01/06/20"},
```

Comment:
I don’t know what TorrentLeech's code base is, but it’s not a Gazelle. On this, I have doubts about other updated information.

Reply:
Its based on Gazelle (Custom Fork) AFAIK.
Torrent Count taken from their website on 01/06/20.
Rest of the information is taken from various reputable online forums.